src/components/templates/Product/Template.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { ProductTemplateHeader } from './Header';
+import { ProductTemplateLoading } from './Loading';
+import { ProductTemplateProgress } from './Progress';
+import { ProductTemplateCard } from './Card';
+import { ZopaFooter } from '../../..';
+
+interface ProductTemplateProps {
+  title: string;
+  subtitle?: string;
+  children: React.ReactNode;
+}
+
+const ProductTemplate = (props: ProductTemplateProps) => {
+  return (
+    <section data-automation="ZA.ProductTemplate">
+      <ProductTemplateHeader title={props.title} subtitle={props.subtitle} />
+      <ProductTemplateContainer> {props.children} </ProductTemplateContainer>
+      <ZopaFooter />
```

Comment:
We could leave Footer out of this. The template is going to be used on a page by page basis and doesn't really make sense to have to re-render the footer multiple times.

Reply:
Makes sense - fair enough